core/domain/exp_services.py

```diff
@@ -205,6 +255,17 @@ def get_multiple_explorations_by_id(exp_ids, strict=True):
     """Returns a dict of domain objects representing explorations with the
     given ids as keys. If an exp_id is not present it is not included in the
     return dict.
+
+    Args: 
+        exp_ids: list of exploration ids.
+
+    Returns:
+        a dict of domain objects representing explorations having given 
```

Comment:
A dict whose keys are exp_ids and whose values are the corresponding Exploration domain objects. The return dict omits exp_ids whose storage models could not be fetched.